src/ConnectedMode/Migration/MSBuildFileCleaner.cs

```diff
@@ -30,15 +30,13 @@ namespace SonarLint.VisualStudio.ConnectedMode.Migration
 {
     [Export(typeof(IFileCleaner))]
     [PartCreationPolicy(CreationPolicy.NonShared)]
-    internal class MSBuildFileCleaner : IFileCleaner
+    internal partial class MSBuildFileCleaner : IFileCleaner
```

Comment:
Why did you make this a partial class? (Not questioning the decision I've just never made a partial class)

Reply:
(answered in separate comment)